doc/api/timers.md

```diff
@@ -23,6 +23,16 @@ running as long as the immediate is active. The `Immediate` object returned by
 [`setImmediate()`][] exports both `immediate.ref()` and `immediate.unref()`
 functions that can be used to control this default behavior.
 
+### immediate.hasRef()
```

Comment:
hmm... maybe `isRef()`? `hasRef` just seems... awkward.

Reply:
isRefed?

Reply:
I don't really mind what is chosen